Gems/EMotionFX/Code/Source/Editor/InspectorBus.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <QtGlobal>
+#include <QString>
+QT_FORWARD_DECLARE_CLASS(QWidget)
+
+namespace EMStudio
+{
+    class InspectorRequests
+        : public AZ::EBusTraits
+    {
+    public:
+        // Call in case a fully customized widget shall be shown in the inspector. The header widget will be shown above the given widget.
+        virtual void Update([[maybe_unused]] const QString& headerTitle, [[maybe_unused]] const QString& iconFilename, [[maybe_unused]] QWidget* widget) {}
```

Comment:
question: Are we sure we want to have default implementations for a request bus? These can be super useful for a notification bus, but usually you want the implementer of a request bus to handle all the requests explicitly. Maybe in this situation it is okay, but it can lead to some confusing bugs if you don't implement everything that should be required